src/Driver/WpcliDriver.php

```diff
@@ -38,11 +43,18 @@ class WpcliDriver extends BaseDriver
      * @param string $path  Absolute path to WordPress site's files. This or $alias must be not falsey.
      * @param string $url   WordPress site URL.
      */
-    public function __construct($alias, $path, $url)
+    public function __construct($alias, $path, $url, $binary = null)
     {
         $this->alias = ltrim($alias, '@');
         $this->path  = realpath($path);
         $this->url   = rtrim(filter_var($url, FILTER_SANITIZE_URL), '/');
+
+        //Support Windows
+        if (is_null( $binary ) && DIRECTORY_SEPARATOR === '\\') {
+            $this->binary = 'wp.bat';
+        } elseif (! is_null( $binary)) {
+            $this->binary = $binary;
```

Comment:
* Let's have the bracket spacing consistent, on lines 53 and 55 (no spaces, unless WordPress).
 * To avoid confusion, this space `! is_null(` is correct and consistent with the rest of the project so far. 👍